components/scroll-lock/test/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { createScrollLockComponent } from '..';
+
+describe( 'scroll-lock', () => {
+	const lockingClassName = 'test-lock-scroll';
+
+	// Use a separate document to reduce the risk of test side-effects.
```

Comment:
There is no risk, all test suites are isolated when you use Jest :)

Reply:
I was very excited by this, but in my testing, it seems like the global `document` persists between tests and suites.

Reply:
Interesting, I will check it once in `master`, out of curiosity :)

Reply:
I added a class to the body in one test, checked for its presence in `beforeEach`, and found the class on the body in the second test. Are you up for letting me know if you find different behavior? I'd love to take advantage of isolated environment.